### PR TITLE
Preamble-BDD-API

### PR DESCRIPTION
**Pull request by jeffschwartz**
case 1. Add a new type definition.
- [x] checked compilation succeeds with `--target es6` and `--noImplicitAny` options.
- [x] has correct [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)
- [x] has a [test file](http://definitelytyped.org/guides/contributing.html#tests) with the suffix of  `-tests.ts` or `-tests.tsx`.

case 2. Improvement to existing type definition.
- documentation or source code reference which provides context for the suggested changes.  url http://api.jquery.com/html .
  - it has been reviewed by a DefinitelyTyped member.

### CODE REVIEW

**Comment by jeffschwartz** [comment]
Btw I am the author of Preamble which is the framework this .d.ts file is for. Is this pull on hold for some reason? I noticed above that the naming convention isn't checked. Could an admin inform me why the name is not correct so I can fix it :)?

File: preamble-ts-dts/preamble-bdd-api.d.ts [hunk 1]
```diff
@@ -0,0 +1,211 @@
+// Type definitions for Preamble-TS-Standalone
+// Project: https://github.com/Preamble-BDD/standalone/
+// Definitions by: Jeffrey Schwartz <https://github.com/jeffschwartz/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference path="../q/Q.d.ts" />
+
+declare interface IMatcher {
```
Comment (vvakame) [inline preamble-ts-dts/preamble-bdd-api.d.ts]: `Matcher`?

**Comment by vvakame** [comment]
http://definitelytyped.org/guides/best-practices.html#ghost-modules
please check this document.

**Comment by vvakame** [comment]
@jeffschwartz yep, please rename `preamble-ts-dts/preamble-bdd-api.d.ts` to `preamble/preamble.d.ts`.

FYI, http://www.typescriptlang.org/docs/handbook/typings-for-npm-packages.html

**Comment by jeffschwartz** [comment]
@vvakame thank you. I will make the changes as per your request. I am going to close out this PR and will submit a new one once the changes are in place.

File: preamble-ts-dts/preamble-bdd-api.d.ts (hunk 1 above)
Comment (jeffschwartz) [inline preamble-ts-dts/preamble-bdd-api.d.ts]: Are you question my use of prefacing the interface with "I" and suggesting that I shouldn't? Personally & IMHO it serves to distinguish between a class and an interface declaration. It is also a common practice in other languages as well. Eagerly awaiting your feedback on this :)

**Pull request closed without merging**